src/index.ts

```diff
@@ -47,42 +46,48 @@ function loadLocation() { // Inspiration from https://stackoverflow.com/question
                     complexityCircles.checked = true;
                     break;
 
+                case 'autofft':
+                    autoFft = true;
+                    break;
+
                 default:
                     { // no-case-declaration
                         const [k, v] = item.split('=');
                         if (v !== null) { // Restriction to valued keys
                             const w = v && decodeURIComponent(v);
                             switch (k) {
                                 case 'pt':
-                                    { // no-case-declaration
+                                    { // no-case-declaration
                                         const [x, y] = w.split(';');
                                         if (x !== null && y !== null)
-                                            addPoint(Number(x), Number(y), false);
+                                            rawPoints.push({ x: Number(x), y: Number(y) });
                                     }
                                     break;
 
                                 case 'range':
-                                    parameterSlider.value = w;
+                                    parameter = Number(w);
                                     break;
 
                                 case 'circles':
                                     complexityCircles.checked = Boolean(Number(w));
                                     break;
 
                                 case 'complexity':
-                                    complexityNumber.value = w;
+                                    complexity = Number(w);
                                     break;
 
                                 case 'fftsize':
+                                    autoFft = false;
                                     fftSize = Number(w);
-                                    fft = new FFT(fftSize);
-                                    input = fft.createComplexArray() as number[];
-                                    output = fft.createComplexArray() as number[];
+                                    break;
+
+                                case 'autofft':
+                                    autoFft = Boolean(Number(w));
                                     break;
                             }
-                        }
+                        }
                     }
-                    break;
+                    break;
```

Comment:
I didn't point these out earlier, sorry. Besides this place, there are two more places lower down shown by GitHub where the line endings are being changed and nothing else. Did `git add --renormalize .` not work for you?

Reply:
I thought I changed those `\r\n` into `\n`. :o

Reply:
It did renomalize, I commited the changes.